ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -273,3 +273,62 @@ ${local.skip_on_weekends}
     action_group = var.action_group_ids
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "batched_uploader_single_failure_detected" {
+  name                = "${var.env}-batched-uploader-single-failure-detected"
+  description         = "QueueBatchedReportStreamUploader failed to successfully complete"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+  severity            = var.severity
+  frequency           = 5
+  time_window         = 5
+  enabled             = contains(var.disabled_alerts, "batched_uploader_single_failure_detected") ? false : true
+
+  data_source_id = var.app_insights_id
+
+  query = <<-QUERY
+dependencies
+${local.skip_on_weekends}
+| where timestamp >= ago(7m) 
```

Comment:
I think all of the previous `timestamp` values were from me, and I made them match the `time_window` because I wasn't sure of the answer to this question and wasn't able to find any documentation either way. I think treating 7 as the correct value makes sense, but I don't think Azure will let you use it - pretty sure the next allowed value after 5 is 10.

Reply:
If the code is to be believed...as long as it's an integer greater than 5, it should pass the validation: https://github.com/hashicorp/terraform-provider-azurerm/blob/298765e3f2433d16b0f15817f69f1e40680509dc/internal/services/monitor/monitor_scheduled_query_rules_alert_resource.go#L236

Reply:
I just ran a test with this, and it looks like 7 minutes is valid! I have updated `time_window` to match.

Reply:
Huh that's interesting - you can't choose that from the console. Good to know!